src/components/ReviewContainer.jsx

```diff
@@ -17,8 +20,39 @@ export const ReviewContainer = () => {
         })
     }, [])
 
+    const filteredReviews = reviews.filter((review) => {
```

Comment:
Instead of filtering the reviews here at the front end, please make use of your API as it should have an endpoint that accepts queries.